arcs/logparser.py

```diff
@@ -40,28 +40,35 @@
 def _convert_type(k, v):
     return TYPE_CONVERTERS.get(k, lambda x: x)(v)
 
+
```

Comment:
Yay! Pep8 is a lot happier now! In fact, if we add

```
[pep8]
max_line_length = 99
```
to a tox.ini in the base folder, all we get now is:

```
./calc_iaa.py:40:100: E501 line too long (132 > 99 characters)
./collection.py:47:81: E502 the backslash is redundant between brackets
./collection.py:48:75: E502 the backslash is redundant between brackets
./collection.py:49:79: E502 the backslash is redundant between brackets
./collection.py:53:81: E502 the backslash is redundant between brackets
./collection.py:54:75: E502 the backslash is redundant between brackets
./collection.py:55:79: E502 the backslash is redundant between brackets
./collection.py:157:100: E501 line too long (135 > 99 characters)
./collection.py:196:26: W601 .has_key() is deprecated, use 'in'
./download_crowdflower.py:103:100: E501 line too long (106 > 99 characters)
./krippendorff_alpha.py:55:100: E501 line too long (135 > 99 characters)
./krippendorff_alpha.py:102:100: E501 line too long (121 > 99 characters)
./krippendorff_alpha.py:140:100: E501 line too long (101 > 99 characters)
./krippendorff_alpha.py:142:100: E501 line too long (135 > 99 characters)
./logparser.py:40:1: E302 expected 2 blank lines, found 1
```